rest/model/patch.go

```diff
@@ -333,33 +371,3 @@ func (g *githubPatch) ToService() (interface{}, error) {
 	res.Author = utility.FromStringPtr(g.Author)
 	return res, nil
 }
-
-type PatchesByStatus []APIPatch
-
-func (p PatchesByStatus) Len() int {
-	return len(p)
-}
-
-func (p PatchesByStatus) Less(i, j int) bool {
-	return p[i].patchStatusPriority() < p[j].patchStatusPriority()
-}
-
-func (p PatchesByStatus) Swap(i, j int) {
-	p[i], p[j] = p[j], p[i]
-}
-
-// patchStatusPriority answers the question of what the patch status should be
```

Comment:
can you move this comment up to the new function?

Reply:
good idea, thanks!